ui/components/multichain/account-overview/account-overview-tabs.tsx

```diff
@@ -109,15 +109,15 @@ export const AccountOverviewTabs = ({
 
   return (
     <Box style={{ flexGrow: '1' }}>
-      <Tabs
-        defaultActiveTabKey={defaultHomeActiveTabName}
+      <Tabs<AccountOverviewTabKey>
```

Comment:
Updated Tabs to allow for generic string type so we can use `AccountOverviewTabKey` enum which values are strings here